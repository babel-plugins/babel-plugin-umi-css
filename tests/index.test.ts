import * as babel from '@babel/core';
import plugin from '@/index';

describe('babel-plugin-umi-css-modules', () => {
  it('work', () => {
    const example = `
      var foo = 1;
      if (foo) console.log(foo);
    `;

    const { code } = babel.transform(example, {
      plugins: [plugin]
    });

    console.log(code);

    expect(code).toMatchSnapshot();
  })
});
