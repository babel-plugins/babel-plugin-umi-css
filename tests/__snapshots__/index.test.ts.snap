// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-plugin-umi-css-modules work 1`] = `
"var bar = 1;
if (bar) console.log(bar);"
`;
